lib/html_sigs.mli

```diff
@@ -1141,6 +1141,7 @@ sig
      and type Xml.event_handler = Xml.event_handler
      and type Xml.mouse_event_handler = Xml.mouse_event_handler
      and type Xml.keyboard_event_handler = Xml.keyboard_event_handler
+     and type Xml.touch_event_handler = Xml.touch_event_handler
```

Comment:
You should need the same thing on the svg side.

Reply:
I have them there – just missed one location.